forktruck:
  # Publish all joint states -----------------------------------
  /joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  /joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: vplate_joint
    pid: {p: 20000.0, i: 10, d: 0.0}
  /joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: fork_joint
    pid: {p: 20000.0, i: 10, d: 0.0}
  /joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: ltooth_joint
    pid: {p: 10000.0, i: 10, d: 0.0}
  /joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: rtooth_joint
    pid: {p: 10000.0, i: 10, d: 0.0}
  /joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: camera_joint
    pid: {p: 10000.0, i: 10, d: 0.0}

